internal/datastore/postgres/postgres.go

```diff
@@ -97,6 +98,14 @@ var (
 	getNow = psql.Select("NOW()")
 
 	tracer = otel.Tracer("spicedb/internal/datastore/postgres")
+
+	supportedQueryExecModes = map[pgx.QueryExecMode]string{
+		pgx.QueryExecModeCacheStatement: "cache_statement",
+		pgx.QueryExecModeCacheDescribe:  "cache_describe",
+		pgx.QueryExecModeDescribeExec:   "describe_exec",
+		pgx.QueryExecModeExec:           "exec",
+		pgx.QueryExecModeSimpleProtocol: "simple_protocol",
```

Comment:
TODO: I need to go through and verify which of these do not work. Some execution modes require type mappings to be registered, so they'll just fail. I'll circle back to this tomorrow.

Reply:
Also write a test for this path.